src/jdk.crypto.ec/share/classes/sun/security/ec/ECPublicKeyImpl.java

```diff
@@ -176,4 +180,20 @@ long getNativePtr() {
         }
         return this.nativeECKey;
     }
-}
+
+    /**
+     * Restores the state of this object from the stream.
+     * <p>
+     * Deserialization of this object is not supported.
+     *
+     * @param  stream the {@code ObjectInputStream} from which data is read
+     * @throws IOException if an I/O error occurs
+     * @throws ClassNotFoundException if a serialized class cannot be loaded
+     */
+    @java.io.Serial
+    private void readObject(ObjectInputStream stream)
+            throws IOException, ClassNotFoundException {
+        throw new InvalidObjectException(
+                "ECPublicKeyImpl keys are not directly deserializable");
+    }
```

Comment:
Please resolve this merge conflict like was done in #272 and for consistency with other versions (e.g. https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/720).